plugins/infinite-sky.cpp

```diff
@@ -0,0 +1,270 @@
+
+#include "Core.h"
+#include "DataDefs.h"
+#include "Debug.h"
+#include "Export.h"
+#include "LuaTools.h"
+#include "PluginManager.h"
+
+#include "modules/World.h"
+#include "modules/EventManager.h"
+
+#include "df/construction.h"
+#include "df/map_block.h"
+#include "df/map_block_column.h"
+#include "df/world.h"
+#include "df/z_level_flags.h"
+
+#include <cstring>
+#include <string>
+#include <vector>
+
+using namespace std;
```

Comment:
```suggestion
using std::string;
using std::vector;
```

Reply:
we discourage `using namespace std` and instead encourage `using` specific individual classes